indra/newview/llvoicewebrtc.cpp

```diff
@@ -301,6 +302,13 @@ void LLWebRTCVoiceClient::updateSettings()
     setRenderDevice(outputDevice);
     F32 mic_level = gSavedSettings.getF32("AudioLevelMic");
     setMicGain(mic_level);
+    
+    llwebrtc::LLWebRTCDeviceInterface::AudioConfig config;
+    config.mEchoCancellation = gSavedSettings.getBOOL("VoiceEchoCancellation");
```

Comment:
If updateSettings() gets called more often than once per frame, you might want to replace getBOOL with LLCachedControl<bool>

Reply:
Hmm, I'm only seeing updateSettings getting called once when we change an audio setting under the debugger.  I'll make the change anyway, as it's the way of the future.